src/main/java/racingcargame/RacingCarGame.java

```diff
@@ -31,6 +31,29 @@ public void startGame(final RandomOption randomOption) {
         }
     }
 
+    public void showWinner() {
+        resultView.printWinner(findWinners());
+    }
+
+    private String findWinners() {
+        cars.sort(Collections.reverseOrder());
+
+        int winnerDistance = cars.get(0).getDistance();
+        String winners = cars.get(0).getCarName();
+        for (int i = 1; i < cars.size(); i++) {
+            winners += selectTie(cars.get(i), winnerDistance);
+        }
+        return winners;
+    }
+
+    private String selectTie(RacingCar car, int winnerDistance) {
+        String tie = "";
+        if(car.isWinner(winnerDistance)) {
+            tie = ", " + car.getCarName();
+        }
+        return tie;
+    }
+
     private void round(RandomOption randomOption) {
         for (RacingCar car : cars) {
             car.tryToMove(randomOption.randomValue());
```

Comment:
아래와 같이도 표현이 가능하답니다 :-) 
```suggestion
               cars.forEach(car -> car.tryToMove(randomOption.randomValue()));
```